server/config/index.js

```diff
@@ -64,6 +65,10 @@ const getTlsConfig = () => {
   };
 };
 
+logger.log(
+  `Auth is ${readConfigSync().auth?.enabled ? 'enabled' : 'disabled'} in config`
```

Comment:
Do we want to check to see if the return value of `readConfigSync()` is `undefined` or is it not necessary because _know_ it will always return a value?

```js
readConfigSync()?.auth?.enabled
```